common/src/main/java/com/wynntils/handlers/inventory/InventoryHandler.java

```diff
@@ -0,0 +1,265 @@
+/*
+ * Copyright © Wynntils 2021-2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.handlers.inventory;
+
+import static net.minecraft.world.inventory.AbstractContainerMenu.QUICKCRAFT_HEADER_CONTINUE;
+import static net.minecraft.world.inventory.AbstractContainerMenu.QUICKCRAFT_HEADER_END;
+import static net.minecraft.world.inventory.AbstractContainerMenu.QUICKCRAFT_HEADER_START;
+import static net.minecraft.world.inventory.AbstractContainerMenu.QUICKCRAFT_TYPE_GREEDY;
+import static net.minecraft.world.inventory.AbstractContainerMenu.getQuickcraftHeader;
+import static net.minecraft.world.inventory.AbstractContainerMenu.getQuickcraftType;
```

Comment:
Please don't use static imports. Afaik spotless should complain, but maybe we couldn't add this as a rule efficiently.